pkg/cgrouprate/cgrouprate.go

```diff
@@ -44,12 +47,19 @@ const (
 	aliveCnt            = 5
 	cleanupInterval     = time.Minute
 	cleanupInactiveTime = time.Minute
+	cgRateMaxEntries    = 32768 // this value could be fine tuned
 )
 
-var (
-	handle     *CgroupRate
-	handleLock sync.RWMutex
-)
+type globalState struct {
+	cgRmdirProg      *program.Program
```

Comment:
does this prog need to be in the state struct? doesn't looks like it's used

Reply:
We can remove it if we want. I just added it because there used to be a global variable with it.

Reply:
right, but looks like only because we initialize it and used it on different places, I'd remove it to prevent future headaches

Reply:
OK, removed. Thanks!